src/codegen/templates/constants.mako

```diff
@@ -0,0 +1,26 @@
+<%
+import utilities.enum_helpers as enum_helpers
+
+enums = enum_helpers.get_enums(data)
+%>\
+# Do not edit this file; it was automatically generated.
+
+from enum import Enum
+
+# Constants
+AUTO = -1
+CFG_DEFAULT = -1
+DEFAULT = -1
+READ_ALL_AVAILABLE = -1
+WAIT_INFINITELY = -1.0
+
+
+# Enums
+% for name, enum_metadata in enums.items():
```

Comment:
@vigkre @MounikaBattu17

`enums.py` now (as of https://github.com/ni/grpc-device-scrapigen/pull/127) contains multiple enums with the same python_name. This template needs to generate the right ones.

```
class ActiveLevel(Enum):
    ABOVE = 10093  #: Pause the measurement or generation while the signal is above the threshold.
    BELOW = 10107  #: Pause the measurement or generation while the signal is below the threshold.


class AngleUnits(Enum):
    DEGREES = 10146  #: Degrees.
    RADIANS = 10273  #: Radians.
    FROM_CUSTOM_SCALE = 10065  #: Units a custom scale specifies. If you select this value, you must specify a custom scale name.


class AngleUnits(Enum):
    DEGREES = 10146  #: Degrees.
    RADIANS = 10273  #: Radians.
    TICKS = 10304  #: Ticks.
    FROM_CUSTOM_SCALE = 10065  #: Units a custom scale specifies. If you select this value, you must specify a custom scale name.


class AngleUnits(Enum):
    DEGREES = 10146  #: Degrees.
    FROM_CUSTOM_SCALE = 10065  #: Units a custom scale specifies. If you select this value, you must specify a custom scale name.


class AngularVelocityUnits(Enum):
    RPM = 16080  #: Revolutions per minute.
    RADIANS_PER_SECOND = 16081  #: Radians per second.
    DEGREES_PER_SECOND = 16082  #: Degrees per second.
    FROM_CUSTOM_SCALE = 10065  #: Units a custom scale specifies. If you select this value, you must specify a custom scale name.
```

Reply:
Update: I think updating `ENUMS_BLACKLIST` and sorting the enums by python_name will take care of this.

Reply:
Updated `ENUMS_BLACKLIST` and logic in `enum_helpers.py` to get desired ouput.